src/types/AuthenticationMiddlewareOptions.ts

```diff
@@ -0,0 +1,13 @@
+import { NextFunction, Request, Response } from 'express';
+import { Logger } from '.';
+
+type AuthenticationMiddlewareOptions = {
+  tokenInfoEndpoint: string,
+  logger?: Logger,
+  onNotAuthenticatedHandler?: (request: Request, resonse: Response, next: NextFunction, logger: Logger) => void,
```

Comment:
as the signature is used several times, might it make sense to define a type?

Reply:
Here I don't agree. We do not repeat that type in the middleware implementation, but we could write one and export it for lib consumers.